test/setup/contract.ts

```diff
@@ -0,0 +1,370 @@
+import * as abi from '@api3/airnode-abi';
```

Comment:
This file is heavily inspired by https://github.com/api3dao/airseeker/blob/main/test/setup/deployment.ts and https://github.com/api3dao/dapi-management/pull/3/files#diff-731d7bf404aa3af378fe6396cbdbd4328ee7f69d5723adec8f2ed5fdb99bb44cR60